core/src/main/scala/sttp/client3/RequestT.scala

```diff
@@ -431,9 +431,11 @@ class SpecifyAuthScheme[U[_], T, -R](hn: String, rt: RequestT[U, T, R], digestTa
   }
 }
 
+// TODO replace with tag
 case class RequestOptions(
     followRedirects: Boolean,
     readTimeout: Duration,
     maxRedirects: Int,
-    redirectToGet: Boolean
-)
\ No newline at end of file
+    redirectToGet: Boolean,
+    binaryFile: Option[String] = None
```

Comment:
I'm not sure what is the purpose of this - do you want to determine if the response should be saved as a file? If so, maybe we can examine `ResponseAs`?